docs/host/host.md

```diff
@@ -21,10 +21,36 @@ Because Harvester is built on top of Kubernetes and uses etcd as its database, t
 
 ## Node Maintenance
 
+In the following scenarios, you plan to migrate/shutdown the workloads from one node and also possibly shutdown the node.
+
+- Replace/add/remove hardware
+
+- Change the network setting
+
+- Troubleshooting
+
+- Remove a node
+
+Harvester provides `Node Maintenance` feature to run a series of checks and operations automatically.
+
 For admin users, you can click **Enable Maintenance Mode** to evict all VMs from a node automatically. It will leverage the `VM live migration` feature to migrate all VMs to other nodes automatically. Note that at least two active nodes are required to use this feature.
```

Comment:
`For admin users,...`

Is the existing text and has been reviewed before.

In this PR, it is not touched, why a change is required?

Reply:
Anyway, I updated per the comments first.

Reply:
> Is the existing text and has been reviewed before.
> In this PR, it is not touched, why a change is required?

1. Mentions of "Node Maintenance" were changed to "Maintenance Mode" for consistency. Users should ideally see just one term for each concept.
2. Sometimes we must rewrite existing text so that it matches the new text and the whole document flows better.
3. Improving the documentation is my responsibility as technical writer. Deciding how to implement the improvements is my prerogative.

cc: @bk201